.github/workflows/pep8.yml

```diff
@@ -9,23 +9,7 @@ on:
       - '*'
 
 jobs:
-  check_skip:
-    runs-on: ubuntu-latest
-    outputs:
-      skip: ${{ steps.result_step.outputs.ci-skip }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - id: result_step
-        uses: mstachniuk/ci-skip@master
-        with:
-          commit-filter: '[skip ci];[ci skip];[skip github]'
```

Comment:
I think the tag `[skip github]` won't work anymore, right ?
I don't think this is very costly though, considered that we are getting rid of a lot of code as a reward, and I don't think anybody was using it...

Reply:
Yes that's true but I agree that it is not worth to keep all this code. And at some point this action will start failing. I think it's enough that we give the example of `[skip ci]` in the readme as well as a link to the other tags that work.